packages/iTwinUI-react/stories/core/ComboBox.stories.tsx

```diff
@@ -497,3 +498,148 @@ WithCustomMessageIcon.args = {
     <StatusMessage startIcon={<SvgCamera />}>This is a message</StatusMessage>
   ),
 };
+
+export const Loading: Story<Partial<ComboBoxProps<string>>> = (args) => {
```

Comment:
Why did you put the examples in ComboBox.stories.tsx rather than making a new story for skeletons? Is ComboBox the only setting where these would be used?

Reply:
It was a users' request for a loading state in ComboBox so it is easier for the user to copy from our stories.